### PR TITLE
kola/tests/kubernetes: timeout install scripts

### PR DESCRIPTION
**Pull request by pbx0**

### CODE REVIEW

File: kola/tests/kubernetes/coreosbasic.go [hunk 1]
```diff
@@ -317,8 +317,16 @@ func runInstallScript(m platform.Machine, script string, options map[string]stri
 		return err
 	}
 
-	if err := session.Wait(); err != nil {
-		return fmt.Errorf("%s", stderr)
+	// timeout script to prevent it looping forever
+	errc := make(chan error, 1)
+	errc <- session.Wait()
```
Comment (mischief) [inline kola/tests/kubernetes/coreosbasic.go]: wouldn't this need to be in a goroutine?

Comment (pbx0) [inline kola/tests/kubernetes/coreosbasic.go]: Doh, yes indeed, I found this testing it.

**Comment by pbx0** [comment]
@mischief PTAL

**Comment by mischief** [comment]
lgtm

**Pull request merged**